zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/DefaultExporterResourceProvider.java

```diff
@@ -10,47 +10,81 @@
 import io.camunda.exporter.config.ExporterConfiguration;
 import io.camunda.webapps.schema.descriptors.IndexDescriptor;
 import io.camunda.webapps.schema.descriptors.IndexTemplateDescriptor;
+import io.camunda.webapps.schema.descriptors.operate.index.*;
+import io.camunda.webapps.schema.descriptors.operate.template.*;
+import io.camunda.webapps.schema.descriptors.tasklist.index.*;
+import io.camunda.webapps.schema.descriptors.tasklist.template.*;
 import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * This is the class where teams should make their components such as handlers, and index/index
  * template descriptors available
  */
 public class DefaultExporterResourceProvider implements ExporterResourceProvider {
 
-  private String operateIndexPrefix;
+  private static final Logger LOG = LoggerFactory.getLogger(DefaultExporterResourceProvider.class);
+  private String indexPrefix;
+  private String exporterId;
 
   @Override
-  public void init(final ExporterConfiguration configuration) {
-    operateIndexPrefix = configuration.getIndex().getPrefix();
+  public void init(final String exporterId, final ExporterConfiguration configuration) {
+    indexPrefix = configuration.getIndex().getPrefix();
```

Comment:
As discussed, let's introduce the field `IndexSettings#tasklistPrefix` and use it for now. And put a TODO item to clean this up when decision on index prefixes topic is done.